Ghidra/Framework/Docking/src/main/java/docking/widgets/DropDownSelectionTextField.java

```diff
@@ -25,9 +25,9 @@
  * 
  * <P><b>Usage note:</b> Typically this text field will not be used directly, but will 
  * instead be used indirectly by way of an editor.
- * If this field is used directly, then the user should use {@link #setSelectedValue(Object)} and
- * {@link #getSelectedValue()} to get and set data on this field, rather than calling 
- * <strike>{@link #setText(String)}</strike> and <strike>{@link #getText()}</strike>.
+ * If this field is used directly, then the user should use <ins>{@link #setSelectedValue(Object)}</ins> and
+ * <ins>{@link #getSelectedValue()}</ins> to get and set data on this field, rather than calling 
+ * <del>{@link #setText(String)}</del> and <del>{@link #getText()}</del>.
```

Comment:
I think the s tag is more correct?  ins and del seem to be designed more for tracking changes.

Reply:
I was initially going to use `<s>` but thought the use of `<ins>` and `<del>` was more appropriate here as it is implying to use <ins>this</ins> rather than <del>this</del>